### ISSUE TITLE
PAC3 player movement not taking effect correctly

### ISSUE THREAD

**Issue by NullLikesNothing** [CapsAdmin/pac3#1214]
Player movement in a vehicle will not take effect correctly when adding, changing or removing the player movement in the editor.
Say you have a player movement with half the normal gravity and your speed is twice as much as normal.
You decide to be in a vehicle and decide to change your gravity to be zero and exit the vehicle. Your gravity is still half.
Changing or removing it while out of the vehicle is fine.

TL;DR player movement doesn't take effect while in a vehicle and you exit it.
